packages/framework/package.json

```diff
@@ -201,6 +205,18 @@
     "next": {
       "optional": true
     },
+    "class-transformer": {
+      "optional": true
+    },
+    "class-validator": {
+      "optional": true
+    },
+    "class-validator-jsonschema": {
+      "optional": true
+    },
+    "reflect-metadata": {
+      "optional": true
+    },
```

Comment:
@rifont should we remove reflect-metadata from the peer-deps and deps also?
technicaly they are only required for development and the user should define it on their server.
So i am not sure how to interpret this :(

Reply:
We should keep reflect-metadata as a peer-dep. The reason is that the package should be treated as isolated for monorepo development purposes to keep it portable, whilst also informing package consumers that it may be required.

Ideally we could communicate conditionality on the class-validators making reflect-metadata required, however that is not currently a supported NPM feature.

So, this approach will do for now!